gradle.properties

```diff
@@ -38,5 +38,8 @@ javaVersion=1.8
 docker_image_default_repo_root=apache
 docker_image_default_repo_prefix=beam_
 
+# supported flink versions
 flink_versions=1.12,1.13,1.14,1.15,1.16
+# supported python versions
+python_versions=3.7,3.8,3.9,3.10
```

Comment:
Added a gradle property here because now `:sdks:python` project need it to define bdist tasks. This also simplifies gradle.properties in python/test-suites a little bit because now it has a single source of truth for supported python version

Reply:
Can we use this one instead? https://github.com/apache/beam/blob/dced7738d6ae8c1d558e6505843caae80e2b4fef/.test-infra/jenkins/PythonTestProperties.groovy#L22

Reply:
Thanks for pointing to this. Not aware of it. This code path is for Jenkins only (`.test-infra/jenkins/`); gradle files works independently with Jenkins (gradle tasks can trigger locally). To reduce to duplicate declaration, refer to `gradle.properties` in Jenkins script should work

Reply:
there is also [beam](https://github.com/apache/beam/tree/master)/[sdks](https://github.com/apache/beam/tree/master/sdks)/[python](https://github.com/apache/beam/tree/master/sdks/python)/[test-suites](https://github.com/apache/beam/tree/master/sdks/python/test-suites)
/gradle.properties 

Would be good to see if we can these version configurations together

Reply:
is https://github.com/apache/beam/blob/master/sdks/python/test-suites/gradle.properties accessible from the context where you need this?

Reply:
The problem is `sdks/python/test-suites/gradle.properties` is not accessible to `sdks/python` gradle project. I understand `test-suites/gradle.properties` as a configuration to manage which python version used in different test. While for bdist it belongs to the python root project (just as sdist). I did not find a source of truth for supported Python versions accessible in the `sdks/python` context.

Reply:
ack, thanks for checking. @AnandInguva FYI one more place to add python versions in the future.

Reply:
Thanks, will experiment to read gradle.properties in jenkins/github actions so reduce the duplicate declarations of py version